packages/core/src/Platform/detectFramework.ts

```diff
@@ -0,0 +1,8 @@
+import { Framework } from './types';
```

Comment:
This file needs the copyright header.

```suggestion
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
import { Framework } from './types';
```